cmd/deploy.go

```diff
@@ -155,24 +165,30 @@ func postDeploymentRequest(l log.Logger, projectName string, namespaceName strin
 				if resp.Ack {
 					// ack for the resource spec
 					if !resp.GetSuccess() {
-						return errors.Errorf("unable to deploy: %s %s", resp.GetResourceName(), resp.GetMessage())
+						return errors.Errorf("unable to deploy resource: %s %s", resp.GetResourceName(), resp.GetMessage())
 					}
 					deployCounter++
-					l.Info(fmt.Sprintf("%d/%d. %s successfully deployed", deployCounter, totalSpecs, resp.GetResourceName()))
+					spinner.SetProgress(deployCounter)
+					if verbose {
+						l.Info(fmt.Sprintf("%d/%d. %s successfully deployed", deployCounter, totalSpecs, resp.GetResourceName()))
+					}
 				} else {
-					// ordinary progress event
-					l.Info(fmt.Sprintf("info '%s': %s", resp.GetResourceName(), resp.GetMessage()))
+					if verbose {
+						// ordinary progress event
+						l.Info(fmt.Sprintf("info '%s': %s", resp.GetResourceName(), resp.GetMessage()))
+					}
 				}
 			}
+			spinner.Stop()
 		}
-		l.Info("deployed resources")
+		l.Info(coloredSuccess("> Resources deployed."))
```

Comment:
Is it intended that we are showing number of jobs deployed but not number of resources deployed?

- Resources deployed.
- Successfully deployed %d/%d jobs

Reply:
Good catch, fixed it.